src/main/resources/templates/challenge.html

```diff
@@ -15,12 +15,185 @@
 		<a href="/dashboard" class="primary"><i class="fa fa-graduation-cap"></i>Submit</a>
 	</div>
 </div>
-<div class="challenge-tri">
-<div><textarea class="challenge-input" id="challenge-input">
-// Challenge: Type something that triggers the pretty print.
-// Try something like "var a = document.getElementById('hi');"
-</textarea></div>
-	<div><pre><code id="challenge-pretty" class="js"></code></pre></div>
+<div class="challenge-dual-pane">
+	<div class="left">
+		<div class="challenge-side-header">
+			<div class="title">Simple Dependency Injection</div>
+			<div class="subtitle">Java Challenge</div>
+		</div>
+		<div class="challenge-side-body">
+			<div class="block">
+				Implement a simple dependency injection library that has a single
+				<code>#provide</code> method. You may assume the following:
+			</div>
+			<ul class="block">
+				<li>All injectable constructors are annotated with <code>@Inject</code>.</li>
+				<li>A class named <code>InjectException</code> exists and extends <code>Exception</code>.</li>
+				<li>Only the core Java JDK is allowed. All versions are allowed.</li>
+			</ul>
+			<div class="block">
+				Any other non-listed assumptions must be handled appropriately.
+			</div>
+		</div>
+	</div>
+	<div class="right">
+		<div>
+			<textarea class="code-highlight" id="challenge-input">import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
```

Comment:
All of the underlining is gone (yay!), but the text alignment issue is still there for my smaller screen on Chrome, even after doing a hard cache refresh. Assuming we'll be using a larger monitor for the demo, and it doesn't affect the demo, though, I think it's fine to ignore the problem for now.

![image](https://cloud.githubusercontent.com/assets/13812345/26491508/cb06cd4e-41c4-11e7-9153-a27dd530809d.png)